src/client/src/actions/patients.js

```diff
@@ -43,6 +43,26 @@ const getPatient = async (id, count) => {
     }
 }
 
+const getAllPatientData = async (id, count) => {    
+    
+    const url = `https://r4.smarthealthit.org/Patient/${id}/$everything?_count=${count}`
```

Comment:
I am pretty sure we are using a different server now. There should also be a route on our server for this that should be used here instead of the direct url.

Reply:
This will be overwritten when we merge #17 so we can push it first.